### ISSUE TITLE
Supported Substitutions section in LTI doc is empty

### ISSUE THREAD

**Issue by bermudezjd** [instructure/canvas-lms#1335]
[This](https://canvas.instructure.com/doc/api/file.tools_variable_substitutions.html) document used to contain a detailed section on supported variable substitution, but it's been blank for at least a few weeks.

**Comment by Mindtoeye** [comment]
It's been gone for much longer now and I kinda need it at the moment.

**Comment by jonespm** [comment]
Yeah, I just noticed this too, the actual document in github still has everything though!

https://github.com/instructure/canvas-lms/blob/master/doc/api/tools_variable_substitutions.md

This looks like where it should be generated from

**Comment by jonespm** [comment]
This looks to have been fixed since last the last comment in 2018.
